examples/queue/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	marathon "github.com/gambol99/go-marathon"
+)
+
+const marathonURL = "http://127.0.0.1:8080"
```

Comment:
Please read the Marathon URL from a flag so that users can configure it, similar to how [some of the other examples](https://github.com/zyfdegh/go-marathon/blob/08cb645a3e99296eb1d2a2171105a0db1de01589/examples/applications/main.go#L30) also do it.